### ISSUE TITLE
Build failure: `libreoffice-fresh` and `liberation-fonts`

### ISSUE THREAD

**Issue by Jesin** [NixOS/nixpkgs#249239]
### Steps To Reproduce

1. build `libreoffice-fresh` from the `nixos-unstable` channel

### Build log

```
these 4 derivations will be built:
  /nix/store/mzqzk89bj3g7q09nirwnyiz2r6lilisr-liberation-fonts-1.07.5.drv
  /nix/store/a6j7czkrdh8ykh88xwkig5h25g148h8d-fonts.conf.drv
  /nix/store/32wdalvr335kc7hg8rclq01ysxq66dl0-libreoffice-7.5.4.1.drv
  /nix/store/5vw00p7b2lis4k2gznyjb0zrdwqwqzc7-libreoffice-7.5.4.1-wrapped.drv
these 10 paths will be fetched (1165.05 MiB download, 1895.73 MiB unpacked):
  /nix/store/pyfq4pnq5d58vf5xlkkh959hgwglcig3-cairo-1.17.6.tar.xz
  /nix/store/xizqh6vr4ly3kfgrzr44qjrqp3c8dkn8-gdb-13.2
  /nix/store/62rn59w6cw8wi57q5hyi9mi29vx0ffl7-gtk+3-3.24.38-dev
  /nix/store/r7qrvm26afymba27d9r0gqzvnx10jwfr-libreoffice-7.5.4.1.tar.xz
  /nix/store/4d450xdwprl2v2ij0rk0r8q4hp318djm-libreoffice-help-7.5.4.1.tar.xz
  /nix/store/0jqlyjxfv7xx9i6la612wc8cqpawjvqr-libreoffice-translations-7.5.4.1.tar.xz
  /nix/store/h23h037l6b9xs7wsqpi024alxk7pki9l-nss-3.88.1-with-nspr-4.35.tar.gz
  /nix/store/1xgwdsm7c6s32yx1za5k74j36sm9apq2-odfvalidator-0.9.0-RC2-SNAPSHOT-jar-with-dependencies-2726ab578664434a545f8379a01a9faffac0ae73.jar
  /nix/store/c58bpllrikf11mybac8qf27r4npgmfh9-openjdk-17.0.7+7
  /nix/store/qj7q9z2zw3jf3yw27jm3srcaihxz0by6-postgresql-13.10.tar.bz2
copying path '/nix/store/pyfq4pnq5d58vf5xlkkh959hgwglcig3-cairo-1.17.6.tar.xz' from 'https://cache.nixos.org'...
copying path '/nix/store/r7qrvm26afymba27d9r0gqzvnx10jwfr-libreoffice-7.5.4.1.tar.xz' from 'https://cache.nixos.org'...
copying path '/nix/store/4d450xdwprl2v2ij0rk0r8q4hp318djm-libreoffice-help-7.5.4.1.tar.xz' from 'https://cache.nixos.org'...
copying path '/nix/store/0jqlyjxfv7xx9i6la612wc8cqpawjvqr-libreoffice-translations-7.5.4.1.tar.xz' from 'https://cache.nixos.org'...
copying path '/nix/store/h23h037l6b9xs7wsqpi024alxk7pki9l-nss-3.88.1-with-nspr-4.35.tar.gz' from 'https://cache.nixos.org'...
copying path '/nix/store/1xgwdsm7c6s32yx1za5k74j36sm9apq2-odfvalidator-0.9.0-RC2-SNAPSHOT-jar-with-dependencies-2726ab578664434a545f8379a01a9faffac0ae73.jar' from 'https://cache.nixos.org'...
copying path '/nix/store/qj7q9z2zw3jf3yw27jm3srcaihxz0by6-postgresql-13.10.tar.bz2' from 'https://cache.nixos.org'...
copying path '/nix/store/xizqh6vr4ly3kfgrzr44qjrqp3c8dkn8-gdb-13.2' from 'https://cache.nixos.org'...
copying path '/nix/store/62rn59w6cw8wi57q5hyi9mi29vx0ffl7-gtk+3-3.24.38-dev' from 'https://cache.nixos.org'...
copying path '/nix/store/c58bpllrikf11mybac8qf27r4npgmfh9-openjdk-17.0.7+7' from 'https://cache.nixos.org'...
building '/nix/store/mzqzk89bj3g7q09nirwnyiz2r6lilisr-liberation-fonts-1.07.5.drv'...
unpacking sources
unpacking source archive /nix/store/mxqm5r24izzn4z41kykiaa20ka17fcl5-source
source root is source
patching sources
updateAutotoolsGnuConfigScriptsPhase
configuring
no configure script, doing nothing
building
build flags: SHELL=/nix/store/ir0j7zqlw9dc49grmwplppc7gh0s40yf-bash-5.2-p15/bin/bash
fontforge -script scripts/fontexport.pe -ttf src/LiberationMono-Regular.sfd src/LiberationMono-Bold.sfd src/LiberationMono-Italic.sfd src/LiberationMono-BoldItalic.sfd src/LiberationSans-Regular.sfd src/LiberationSans-Bold.sfd src/LiberationSans-Italic.sfd src/LiberationSans-BoldItalic.sfd src/LiberationSerif-Regular.sfd src/LiberationSerif-Bold.sfd src/LiberationSerif-Italic.sfd src/LiberationSerif-BoldItalic.sfd
Copyright (c) 2000-2023. See AUTHORS for Contributors.
 License GPLv3+: GNU GPL version 3 or later <http://gnu.org/licenses/gpl.html>
 with many parts BSD <http://fontforge.org/license.html>. Please read LICENSE.
 Version: 20230101
 Based on sources from 2023-01-01 00:00 UTC-D.
Generating ttf from src/LiberationMono-Regular.sfd
Generating ttf from src/LiberationMono-Bold.sfd
Generating ttf from src/LiberationMono-Italic.sfd
Generating ttf from src/LiberationMono-BoldItalic.sfd
Generating ttf from src/LiberationSans-Regular.sfd
Generating ttf from src/LiberationSans-Bold.sfd
Generating ttf from src/LiberationSans-Italic.sfd
Generating ttf from src/LiberationSans-BoldItalic.sfd
Lookup subtable contains unused glyph afii10066.alt1 making the whole subtable invalid
Lookup subtable contains unused glyph afii10068.alt1 making the whole subtable invalid
Lookup subtable contains unused glyph afii10069.alt1 making the whole subtable invalid
Lookup subtable contains unused glyph afii10081.alt1 making the whole subtable invalid
Lookup subtable contains unused glyph afii10084.alt1 making the whole subtable invalid
Lookup subtable contains unused glyph afii10090.alt1 making the whole subtable invalid
Generating ttf from src/LiberationSerif-Regular.sfd
Generating ttf from src/LiberationSerif-Bold.sfd
Note: On Windows many apps can have problems with this font's kerning, because 1 of its glyph kern pairs cannot be mapped to unicode-BMP kern pairs (eg, they have a Unicode value of -1) To avoid this, go to Generate, Options, and check the "Windows-compatible 'kern'" option.
Generating ttf from src/LiberationSerif-Italic.sfd
Note: On Windows many apps can have problems with this font's kerning, because 1 of its glyph kern pairs cannot be mapped to unicode-BMP kern pairs (eg, they have a Unicode value of -1) To avoid this, go to Generate, Options, and check the "Windows-compatible 'kern'" option.
Generating ttf from src/LiberationSerif-BoldItalic.sfd
Note: On Windows many apps can have problems with this font's kerning, because 4 of its glyph kern pairs cannot be mapped to unicode-BMP kern pairs (eg, they have a Unicode value of -1) To avoid this, go to Generate, Options, and check the "Windows-compatible 'kern'" option.
mkdir -p liberation-fonts-ttf-1.07.5
mv src/LiberationMono-Regular.ttf src/LiberationMono-Bold.ttf src/LiberationMono-Italic.ttf src/LiberationMono-BoldItalic.ttf src/LiberationSans-Regular.ttf src/LiberationSans-Bold.ttf src/LiberationSans-Italic.ttf src/LiberationSans-BoldItalic.ttf src/LiberationSerif-Regular.ttf src/LiberationSerif-Bold.ttf src/LiberationSerif-Italic.ttf src/LiberationSerif-BoldItalic.ttf liberation-fonts-ttf-1.07.5
installing
install: cannot stat 'README': No such file or directory
error: builder for '/nix/store/mzqzk89bj3g7q09nirwnyiz2r6lilisr-liberation-fonts-1.07.5.drv' failed with exit code 1;
       last 10 log lines:
       > Generating ttf from src/LiberationSerif-Bold.sfd
       > Note: On Windows many apps can have problems with this font's kerning, because 1 of its glyph kern pairs cannot be mapped to unicode-BMP kern pairs (eg, they have a Unicode value of -1) To avoid this, go to Generate, Options, and check the "Windows-compatible 'kern'" option.
       > Generating ttf from src/LiberationSerif-Italic.sfd
       > Note: On Windows many apps can have problems with this font's kerning, because 1 of its glyph kern pairs cannot be mapped to unicode-BMP kern pairs (eg, they have a Unicode value of -1) To avoid this, go to Generate, Options, and check the "Windows-compatible 'kern'" option.
       > Generating ttf from src/LiberationSerif-BoldItalic.sfd
       > Note: On Windows many apps can have problems with this font's kerning, because 4 of its glyph kern pairs cannot be mapped to unicode-BMP kern pairs (eg, they have a Unicode value of -1) To avoid this, go to Generate, Options, and check the "Windows-compatible 'kern'" option.
       > mkdir -p liberation-fonts-ttf-1.07.5
       > mv src/LiberationMono-Regular.ttf src/LiberationMono-Bold.ttf src/LiberationMono-Italic.ttf src/LiberationMono-BoldItalic.ttf src/LiberationSans-Regular.ttf src/LiberationSans-Bold.ttf src/LiberationSans-Italic.ttf src/LiberationSans-BoldItalic.ttf src/LiberationSerif-Regular.ttf src/LiberationSerif-Bold.ttf src/LiberationSerif-Italic.ttf src/LiberationSerif-BoldItalic.ttf liberation-fonts-ttf-1.07.5
       > installing
       > install: cannot stat 'README': No such file or directory
       For full logs, run 'nix log /nix/store/mzqzk89bj3g7q09nirwnyiz2r6lilisr-liberation-fonts-1.07.5.drv'.
error: 1 dependencies of derivation '/nix/store/a6j7czkrdh8ykh88xwkig5h25g148h8d-fonts.conf.drv' failed to build
```

### Notify maintainers

@7c6f434c

### Metadata

```console
[jesin@jesin-nixos00:~]$ nix-shell -p nix-info --run 'nix-info -m'
 - system: `"x86_64-linux"`
 - host os: `Linux 6.1.45, NixOS, 23.11 (Tapir), 23.11pre514192.6e287913f7b1`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.17.0`
 - channels(root): `"home-manager, nixos"`
 - channels(jesin): `""`
 - nixpkgs: `/nix/var/nix/profiles/per-user/root/channels/nixos`
```

**Comment by eclairevoyant** [comment]
not related to libreoffice; font package build fixed in #249059

**Comment by Jesin** [comment]
Ah thanks. I already searched for `liberation-fonts` and #249059 didn't show up, but it shows up if I just search for `liberation`. I guess in the future I'll try to relax my search terms a bit.
